src/Reflection/SignatureMap/functionMap.php

```diff
@@ -4593,15 +4594,19 @@
 'imageftbbox' => ['array', 'size'=>'float', 'angle'=>'float', 'font_file'=>'string', 'text'=>'string', 'extrainfo='=>'array'],
 'imagefttext' => ['array', 'im'=>'resource', 'size'=>'float', 'angle'=>'float', 'x'=>'int', 'y'=>'int', 'col'=>'int', 'font_file'=>'string', 'text'=>'string', 'extrainfo='=>'array'],
 'imagegammacorrect' => ['bool', 'im'=>'resource', 'inputgamma'=>'float', 'outputgamma'=>'float'],
-'imagegd' => ['bool', 'im'=>'resource', 'filename='=>'string|resource|null'],
-'imagegd2' => ['bool', 'im'=>'resource', 'filename='=>'string|resource|null', 'chunk_size='=>'int', 'type='=>'int'],
+'imagegd' => ['bool', 'im'=>'resource', 'filename='=>'string|resource'],
+'imagegd\'1' => ['string', 'im'=>'resource', 'filename='=>'null'],
```

Comment:
Can you test that ParametersAcceptorSelector will figure this out? Add a new data provider to NodeScopeResolverTest::testFileAsserts and see that the right imagegd() calls lead to bool or string with assertType(), thanks.